praetorian_cli/ui/console/commands/context.py

```
@@ -197,6 +226,45 @@ def _show_detail_by_number(self, num: int):
+                    detail = entry['content'][:120] if entry['content'] else ''
+                table.add_row(str(i), 'call', name, detail)
+            elif entry['role'] == 'tool response':
+                name = entry.get('name', '')
+                summary = entry.get('summary', '')
+                detail = summary if summary else ''
+                if not detail:
+                    detail = entry['content'][:120] + '...' if len(entry['content']) > 120 else entry['content']
+                table.add_row(str(i), 'response', name, detail)
+
+        self.console.print(table)
+        self.console.print(f'[dim]Use "set verbose on" to see full details inline during Marcus queries.[/dim]')
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)